apps/mobile/i18n.js

```diff
@@ -1,4 +1,4 @@
-import i18next from 'i18next'
+import i18n from '@hylo/shared/i18n'
```

Comment:
It's not expected that this i18n as shared module stuff is coming in here, it is already merged and once it went to dev I merged it up to your branch with this is based upon... Anyway, you never got to review this bit so no harm but this PR was a bit shorter without this extra.